sdk/src/test/java/com/mapbox/maps/extension/observable/MapboxMapTest.kt

```diff
@@ -48,6 +44,37 @@ class MapboxMapTest {
       size = 181576,
       error = Error("not-found", "error message")
     )
+    val requestMap = hashMapOf(
+      Pair("loading-method", Value(listOf(Value("network")))),
+      Pair("url", Value("https://api.mapbox.com")),
+      Pair("kind", Value("tile")),
+      Pair("priority", Value("regular"))
+    )
+    val responseMap = hashMapOf(
```

Comment:
Where does this come from?

Reply:
We need to create a `Value` data, in the next step` event.getResourceEventData()` will parse this part

Reply:
Did you just take it out of your head? 
Cause I did not see similar code removed in this PR..

Reply:
Oh, it's in a deleted ShadowValueConverter.

Reply:
would it be easier to parse from a json string instead of constructing the Value class manually?

Reply:
Yeah, but since it works now, let's keep it.